anvil/src/main/kotlin/dev/inkremental/KAnvil.kt

```diff
@@ -25,11 +25,18 @@ inline fun <reified T: View> v(noinline r: () -> Unit = {}) = v(T::class, r)
 
 inline fun <reified T: View, reified S: ViewScope> v(s: S, noinline r: S.() -> Unit = {}) = v(T::class, r.bind(s))
 
+inline fun <reified S: ViewScope>xml(@LayoutRes layoutId: Int, s: S, noinline r: S.() -> Unit = {}) {
+    start(layoutId)
+    r.bind(s).invoke()
+    end()
+}
+
```

Comment:
Let's reuse unscoped `xml` function we already have:
```suggestion
inline fun <reified S: ViewScope> xml(@LayoutRes layoutId: Int, s: S, noinline r: S.() -> Unit = {}) = xml(layoutId, r.bind(s))
```
This way we don't need to generalize `start(Int)` and avoid redundant duplication.

Reply:
I'm also not fond of public `end` and `skip`. @sgrekov do you think it's reasonable to move these to extensions of `Inkremental` object so that they don't pollute global scope? (This, of course, would be for another PR)

Reply:
For start() and end() agree, let's hide them. But for hide() I would not hide it, at least for now, because it is used as a public API with SwipeRefreshLayout.